applications/crossbar/doc/push_notification_subscriptions.md

```diff
@@ -0,0 +1,64 @@
+### Push Notification Subscriptions
+
+#### About Push Notification Subscriptions
+
```

Comment:
Can you add some text here so folks looking at this doc get an idea of what the API is useful for?

Reply:
Added a brief description of the purpose that this crossbar endpoint serves.